docs/src/app.tsx

```diff
@@ -6,6 +6,7 @@ import { mergeThemes } from 'src/lib'
 import { ThemeContext, ThemeContextData, themeContextDefaults } from './context/ThemeContext'
 import Routes from './routes'
 import { PerfDataProvider } from './components/ComponentDoc/PerfChart'
+import Debug from '../../packages/react/src/components/Debug/Debug'
```

Comment:
```suggestion
import { Debug } from '@stardust-ui/react'
```

Should it be exported from `@stardust`?